fontforgeexe/fontinfo.c

```diff
@@ -2657,7 +2657,7 @@ static int CheckNames(struct gfi_data *d) {
     buts[0] = _("_OK"); buts[1] = _("_Cancel"); buts[2]=NULL;
 
     if ( u_strlen(ufont)>63 ) {
-	ff_post_error(_("Bad Font Name"),_("A PostScript name should be ASCII\nand must not contain (){}[]<>%%/ or space\nand must be shorter than 63 characters"));
+	ff_post_error(_("Bad Font Name"),_("A Fontname should be ASCII\nand must not contain (){}[]<>%%/ or space\nand must be shorter than 63 characters"));
```

Comment:
> Yes, but the person who types an invalid character there probably does not know that this applies to all fields in the window so this is extra information.

An error message is no place for "extra information" it should tell the user what he/she did wrong as short and precise as possible. 
If you absolutely want to educate your users in error messages then you should make it clear that you are doing so e.g. 
```
Fontname should be ASCII\nand must not contain (){}[]<>%%/ or space\nand must be shorter than 63 characters.\nThis is true for all Postscript names.
```